addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.js

```diff
@@ -112,6 +114,24 @@ export class TicketScreen extends Component {
             BACKSPACE,
         ]);
     }
+    async onClickScanOrder(qrcode) {
+        if (qrcode) {
+            const ref = new URL(qrcode).searchParams.get("order_uuid");
+            const order = await this.pos.models["pos.order"].getBy("uuid", ref);
```

Comment:
@caburj 
Did you mean this?
Please let me know if I misunderstood anything

Thanks in advance :+1:

Reply:
No. You'll have to do `this.pos.data.search` if `this.pos.models["pos.order"].getBy("uuid", ref)` didn't return anything.

Reply:
I have updated the same
Could you please have a look.